resources/views/vendor/samlidp/metadata.blade.php

```diff
@@ -0,0 +1,21 @@
+@php echo '<?xml version="1.0"?>' @endphp
+<md:EntityDescriptor xmlns:md="urn:oasis:names:tc:SAML:2.0:metadata" validUntil="2027-01-20T19:04:25Z" cacheDuration="PT1485371065S" entityID="{{ url(config('samlidp.issuer_uri')) }}">
```

Comment:
Questions: What are the "validUntil" and "cacheDuration" values based on when they are generated? Can't current "hardcoded" values be generated in the code?

Reply:
These variables mean
- `validUntil` the expiration time of the metadata
- `cacheDuration` the maximum period of time a consumer should cache the metadata

I added some functions to generate them dynamically